README.md

```diff
@@ -0,0 +1,224 @@
+
+[![torch-ort](https://img.shields.io/pypi/v/torch-ort)](https://pypi.org/project/torch-ort/)
+[![python](https://img.shields.io/badge/python-3.7%2B-blue)]()
+[![pytorch](https://img.shields.io/badge/pytorch-1.12.1%2B-blue)]()
+[![API Checks](https://github.com/pytorch/ort/actions/workflows/api-ci.yml/badge.svg)](https://github.com/pytorch/ort/actions/workflows/api-ci.yml)
+[![Docs](https://github.com/pytorch/ort/actions/workflows/doc-automation.yml/badge.svg)](https://github.com/pytorch/ort/actions/workflows/doc-automation.yml)
+
+</div>
+
+---
+
+# Introduction
+
+A library for accelerating PyTorch models using ONNX Runtime:
+
+- torch-ort to train PyTorch models faster with ONNX Runtime
+- moe to scale large models and improve their quality
+- torch-ort-infer to perform inference on PyTorch models with ONNX Runtime and Intel OpenVINO
+
+# 🚀 Installation
+
+## Training
+
+### Pre-requisites
+
+You need a machine with at least one NVIDIA or AMD GPU to run ONNX Runtime for PyTorch.
+
+You can install and run torch-ort in your local environment, or with [Docker](torch_ort/docker/README.md).
+
+### Install in a local Python environment
+
+1. Install CUDA
+
+2. Install CuDNN
+
+3. Install torch-ort
+
+    - `pip install torch-ort`
+
+4. Run post-installation script for ORTModule
+
+    - `python -m torch_ort.configure`
+
+Get install instructions for other combinations in the `Get Started Easily` section at <https://www.onnxruntime.ai/> under the `Optimize Training` tab.
+
+### Verify your installation
+
+1. Clone this repo
+
+    - `git clone git@github.com:pytorch/ort.git`
+
+2. Install extra dependencies
+
+    - `pip install wget pandas sklearn transformers`
+
+3. Run a test training script
+
+    - `python ./ort/tests/bert_for_sequence_classification.py`
+
+## Mixture Of Experts
+
+Mixture of Experts layer implementation is available in the ort_moe folder.
+
+
+Clone this repo
+
+```bash
+git clone https://github.com/pytorch/ort.git
+```
+
+Build MoE
+
+```bash
+cd ort_moe
+pip install build # Install PyPA build
+python -m build
+```
+
+## Inference
+
+### Prerequisites
+
+- Ubuntu 18.04, 20.04
+- Python* 3.7, 3.8 or 3.9
+
+### Install in a local Python environment
+
+- `pip install torch-ort-infer[openvino]`
+- Run post installation configuration script `python -m torch_ort.configure`
+
+### Verify your installation
+
+1. Clone this repo
+
+    - `git clone git@github.com:pytorch/ort.git`
+
+2. Install extra dependencies
+
+    - `pip install wget pandas sklearn transformers`
+
+3. Run a test script
+
+    - `python ./torch_ort_inference/tests/bert_for_sequence_classification.py`
+
+# 📈 Training
+
+The torch-ort library accelerates training of large transformer PyTorch models to reduce the training time and GPU cost with a few lines of code change. It is built on top of highly successful and proven technologies of ONNX Runtime and ONNX format and includes the ONNX Runtime Optimizer and Data Sampler.
```

Comment:
Can we add back the example usage for FusedAdam optimizer, DataSampler etc...

Reply:
@AdamLouly is doing this as we speak